docs/iris/src/developers_guide/deprecations.rst

```diff
@@ -23,9 +23,13 @@ deprecation is accompanied by the introduction of a new public API.
 
 Under these circumstances the following points apply:
 
- - Using the deprecated API must result in a concise deprecation
-   warning.
- - Where possible, your deprecation warning should include advice on
+ - Using the deprecated API must result in a concise deprecation warning which
+   is an instance of :class:`iris._deprecation_helpers.IrisDeprecation`.
+   Much the simplest way is to call
```

Comment:
? Did you object to the meaning or to the wording ?
If the latter, I suppose we could just say "It is simplest to call .."